heliopy/data/util.py

```diff
@@ -800,7 +809,9 @@ def cdf2df(cdf, index_key, dtimeindex=True, badvalues=None,
         else:
             keys[cdf_key] = cdf_key
     # Remove index key, as we have already used it to create the index
-    keys.pop(index_key)
+    # Keep the index key if it wasn't the standard 'Epoch'
+    if index_key == 'Epoch':
+        keys.pop(index_key)
```

Comment:
Why keep the key if it isn't the standard Epoch? The idea behind this line is not to have the index data appearing as both the index and a data column.

Reply:
SEPT data has two epoch fields, Epoch_NS and Epoch_E, one of which corresponds to part of the data (NS detector tubes), the other to the rest. The fields seem to have same values but I don't know if it is quaranteed... We could drop the one copied to the index but would it be clear to the user? That's why I kept the non-standard field, and converted them separately to datetime in stereo.py lines 117-118